.release-please-manifest.json

```
@@ -1,3 +1,4 @@
 {
-  ".": "0.10.4"
+  ".": "0.10.4",
+  "./crates/source-resolver": "1.1.4"
```

```suggestion
  "crates/source-resolver": "1.1.4"
```
Needs to be the same as the path in the config file